### PR TITLE
Repro problem with fragmented GIOP messages

### PR DESCRIPTION
**Pull request by j18ter**
This is a test to reproduce issue #1201. Not sure what needs to
be done to integrate this with the rest of the build, if anything.

Note, that I'm referencing a utility file from the examples
directory, not sure if this is frowned upon?

A free-standing variant of this test, which I was able to build and
test is here: [TAO Fragmentation Bug](https://github.com/NetAcquire/tao_fragmentation_bug)

### CODE REVIEW

**Comment by jwillemsen** [comment]
I would prefer not to use an example file from a completely different location

File: TAO/tests/GIOP_Fragments/Big_String_Sequence/echo.mpc [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// -*- MPC -*-
+project(*idl): taoidldefaults {
+  IDL_Files {
+    Echo.idl
+  }
+  custom_only = 1
+}
+
+project(*Server): taoserver, namingexe, iortable, utils, avoids_corba_e_micro {
```
Comment (jwillemsen) [inline TAO/tests/GIOP_Fragments/Big_String_Sequence/echo.mpc:9]: base projects need cleanup, way too much looking briefly at the test code

Comment (j18ter) [reply TAO/tests/GIOP_Fragments/Big_String_Sequence/echo.mpc]: The only base project I was able to remove without causing the build to fail was `avoids_corba_e_micro`. Note,
this whole example is just a copy of `examples/Simple/echo` with some tweaks to reproduce this bug. We don't
use MPC in our own projects and I don't know enough about MPC to do more than copy existing stuff with small
and obvious adjustments. I've pushed a change to remove  `avoids_corba_e_micro`.

**Comment by j18ter** [comment]
> I would prefer not to use an example file from a completely different location

Unfortunately, this `Simple_util.cpp` file can't be dropped without completely rewriting the application. It could
be redundantly checked-in in the project directory, but this seems worse to me.

File: TAO/tests/GIOP_Fragments/Big_String_Sequence/server.cpp [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#include "../../../examples/Simple/Simple_util.h"
+#include "Echo_i.h"
+
+// This is the main driver program for the time and date server.
```
Comment (jwillemsen) [inline TAO/tests/GIOP_Fragments/Big_String_Sequence/server.cpp:4]: old comment

**Comment by jwillemsen** [comment]
Please don't use std namespace, it isn't fully supported on all ACE/TAO supported platforms

**Comment by jwillemsen** [comment]
Related to #1239

**Pull request merged**